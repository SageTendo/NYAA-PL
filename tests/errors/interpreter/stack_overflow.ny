kawaii func() => {
    func()
}

uWu_nyaa() => func();