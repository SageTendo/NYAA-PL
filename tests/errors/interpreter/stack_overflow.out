Non-kawaii recursion depth exceeded